diff --git a/node_modules/@react-native/gradle-plugin/build.gradle.kts b/node_modules/@react-native/gradle-plugin/build.gradle.kts
index aba3649..ba5a81d 100644
--- a/node_modules/@react-native/gradle-plugin/build.gradle.kts
+++ b/node_modules/@react-native/gradle-plugin/build.gradle.kts
@@ -11,7 +11,7 @@ import org.gradle.configurationcache.extensions.serviceOf
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  kotlin("jvm") version "1.7.22"
+  kotlin("jvm") version "1.9.22"
   id("java-gradle-plugin")
 }
 
@@ -57,8 +57,8 @@ java {
 tasks.withType<KotlinCompile> {
   kotlinOptions {
     jvmTarget = JavaVersion.VERSION_11.majorVersion
-    apiVersion = "1.5"
-    languageVersion = "1.5"
+    apiVersion = "1.9"
+    languageVersion = "1.9"
   }
 }
 
